CMakeLists.txt

```diff
@@ -122,11 +90,13 @@ find_library(libuv_shared NAMES uv libuv PATHS ${LIBUV_DEPS_DIR}/src PATH_SUFFIX
 # Referenced directories and targets
 #
 
-if(${DOXYGEN_FOUND})
-    add_subdirectory(${PROJECT_DOCS_DIR})
-endif(${DOXYGEN_FOUND})
+if(DOXYGEN_FOUND)
+    add_subdirectory(docs)
+endif()
 
-add_subdirectory(${PROJECT_TEST_DIR})
+if(BUILD_TESTING)
```

Comment:
I suspect `BUILD_TESTING` isn't available with `enable_testing`. Anyway this isn't a good reason to switch to `include(CTest)`, for that variable isn't the sole side effect of such a change. I'm sorry.